recipes/rhizopus/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rhizopus" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jwergieluk/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5a92eec05c8e9e98b842ffaf1be70bdd2b11d488b1786c0c77f54f08effd9762
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - rhizopus
```

Comment:
A plain `import rhizopus` doesn't do anything because `__init__.py` is empty. Could you add some imports that better resemble actual use to verify the package isn't broken (e.g. missing dependencies)?

Reply:
I added import statements for all the modules in the package. Also, FWIW I have a `pip check` call now. This package, by design, has no external dependencies.
@dopplershift Do you think it's OK now?

Reply:
Looks good, thanks!